apps/site/src/app/(home)/sections/FAQ/FAQ.tsx

```diff
@@ -18,14 +18,38 @@ const FAQ = async () => {
 
 	return (
 		<section className={styles.container}>
+			<AnteaterFloatie
+				src={leftAnteater}
+				alt="left anteater on floatie"
+				className={styles["left-anteater"]}
+			/>
+
 			<div className={styles.faq}>
-				<div className={styles["vertical-line"]} />
-				<Image src={star} alt="star" className={styles["star-lg"]} />
-				<Image src={star} alt="star" className={styles["star-sm"]} />
-				<Image src={eraser} alt="eraser" className={styles["eraser"]} />
-				<h2 className={styles.title}>FAQ</h2>
+				<div
+					className={`${styles["accordion-border"]} ${styles["answer-body"]} ${styles["header-body"]}`}
+				>
+					<h4>
```

Comment:
Chore: heading elements should be used to specify document structure, not as a shortcut for styling. We can keep the plain "FAQ" as `<h2>` but make it [visually hidden](https://getbootstrap.com/docs/5.3/helpers/visually-hidden/). For the actual text, you can use the corresponding utility classes from Bootstrap, e.g. `className="h4"` but use a regular `div` or `span` instead of a heading element.